WcaOnRails/app/models/competition.rb

```diff
@@ -1718,4 +1718,11 @@ def to_ics
     cal.publish
     cal
   end
+
+  def exempt_from_wca_dues?
+    world_or_continental_championship = (championships.map(&:championship_type) & Championship::MAJOR_CHAMPIONSHIP_TYPES).any?
```

Comment:
Mathematically elegant approach, but not the easiest to read.
What about something along the lines of `championships.map(&:championship_type).any? { |ct| Championship::MAJOR_CHAMPIONSHIP_TYPES.include?(ct) }`?

Reply:
Good call. Updated!